src/sassVarsLoader.test.js

```diff
@@ -101,6 +102,46 @@ describe('With invalid file', () => {
   expectError(`Invalid file: "~invalid~". Consider using "path.resolve" in your config.`)
 })
 
+describe('With single post-processing', () => {
+  beforeAll(async () => {
+    await setup({
+      callback: transformKeys.toKebab,
+      vars: {
+        valueToTransform: 'foo',
+        nested: {
+          works: {
+            Complete: true,
+            veryWellResult: true,
+            withoutProblems: 'indeed',
+          },
+        },
+      },
+    })
+  })
+  expectCorrectResult()
+  expectMarksItselfAsCacheable()
+})
+
+describe('With multi post-processing', () => {
+  beforeAll(async () => {
+    await setup({
+      callback: [transformKeys.toKebab, transformKeys.toUpper],
```

Comment:
While I see your use case and I highly appreciate your PR, I'm not sure it's ideal to bundle certain transform functions with this lib, as this makes the lib a bit complex.

But I like having a transform option. So how about letting the user specify custom transform functions?

In that case I'd call that option something like `transform` instead of `callback` to be a bit more specific:

`transform: userProvidedTransfomFunction`

I see that your use case was transforming the keys, so maybe it can be helpful to provide an option to transform keys specifically, in which case the option could be called something like `transformKeys` or `transformVarNames`:

`transformKeys: _.camelCase`

In this example I simply passed a lodash String transformation function. I think it's better not to re-implement and bundle them with this lib.

What do you think?

Reply:
Thank you for replay, @epegzz. Yes, I agree to exclude transform functions from the lib, because it really looks like overhead ;)

But about option attribute name, I think that `transformKeys` is mush better.
What is next? Should I remove all except new option functionality? How should I write the tests for it without ready-to-use transforming functions?

Reply:
> Should I remove all except new option functionality?
Yeah 👍 

> How should I write the tests for it without ready-to-use transforming functions?
How about just passing a dummy inline transformation function in your test:
```js
await setup({
  transformKeys: key => `transformed-${key}`,
  ...
})
```

Reply:
Looks great! Give me a minute! :)

Reply:
Ok, @epegzz! I have changed code base and ready to continue discussion with new commit :)